### PR TITLE
Added BlogPage

### LINKED ISSUE #50: UI for blog page
Follow CSS BEM architecture.
Make a separate SCSS file for this page.
Do not compile scss.

### PR DESCRIPTION
**Pull request by Shubham-1105**
fixes #50 

I am getting issue in removing the padding between the grid containers

Screenshot (458)
Screenshot (459)

**Addition Information (if any)** <br/>
Mention any additional information that has been taken note of while making the PR, or needs to be taken note of while reviewing the PR. Ignore if none.

### CODE REVIEW

File: frontend/src/pages/BlogPage.jsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import Facebook from "../images/fb-icon.png";
+import Twitter from "../images/twitter-icon.png";
+import ProfilePic from "../images/rashmeet.png";
+import BlogImage from "../images/placeholder-images/blog_img.png";
+
+class BlogPage extends Component{
+   render(){
+    return(
+    <>
+            <div style={{ background: "#FFFDF6" }}>
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:15]: style is not required as you have already given it in css

File: frontend/src/pages/BlogPage.jsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+          <div className="hero">
+            <div className="grid-container fluid">
+              <div className="grid-x ">
+              <div className="cell small-6">
+                  <img src={BlogImage}  className="hero_blogImage"alt="Blog"/>
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:20]: This is not as per design.

File: frontend/src/pages/BlogPage.jsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                        <div className="cell small-6">
+                          <img src={ProfilePic} className="hero__profile" alt="profile"/>
+                        </div>
+                        </div> 
+                      
+
+                    </div>
+                    {/* Author and Date */}
+
+                    <div className="grid-container">
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:42]: x-y grid should not be used here. Use flex instead.

File: frontend/src/pages/BlogPage.jsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                          <h3>Rashmeet Kaur</h3>
+                        </div>
+                        <div className="cell medium-4">
+                          <p>18 May 2020</p>
+                        </div>
+                      </div>
+                    </div>
+                     {/* Social Media Icons */}
+                    <div className="grid-container">
+                      <div className="grid-x">
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:54]: x-y grid should not be used here. Use flex

File: frontend/src/pages/BlogPage.jsx [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+     
+
+                
+                </div>
+               
+              </div>
+            </div>
+
+            <div className="main">
+              <div className="grid-conatiner fluid">
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:88]: Remove the `fluid` class

File: frontend/src/pages/BlogPage.jsx [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+                <div className="grid-y">
+                  <div className="cell large-8">
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:90]: Use `large offset` to keep the article section in the center.

File: frontend/src/pages/BlogPage.jsx [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+                  <div className="grid-container">
+                    <p className="main__para__head">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                                    incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud 
+                                    exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                    </p>
+                  </div>
+
+                  <div className="grid-container">
```
Comment (salil-naik) [inline frontend/src/pages/BlogPage.jsx:99]: `grid-container` class should only be used once. You have used it almost everywhere.

File: frontend/src/css/pages/BlogPage.scss [hunk 8]
```diff
@@ -0,0 +1,41 @@
+@import "../settings/global";
+.hero {
+  background-color: $yellow-bg;
+  padding:80px 0; 
+    
+  
+
+    &__blogImage{
+        border: 10px solid black;
```
Comment (salil-naik) [inline frontend/src/css/pages/BlogPage.scss:9]: border is not required

File: frontend/src/css/pages/BlogPage.scss [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+  
+ 
+  }
+
+  .main{
+    padding: 100px;
+    &__para__head{
+      color:#4A2343;
+      font-weight: bolder;
```
Comment (salil-naik) [inline frontend/src/css/pages/BlogPage.scss:36]: There are scss classes made for font-weight and colours

**Review by salil-naik — CHANGES_REQUESTED** [review]
There is no need to use grid-y.  Every block element by default comes one below another. 

- [ ] Do not use grid-y anywhere
- [ ] Rewrite the code as you have taken `grid-container` and `grid-y` everywhere. `grid-container` is required only twice for this page. One for the top part, and one for the article. Remove all others.
- [ ] Run scss and jsx through [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). 
- [ ] Use avatar component
- [ ] Follow the grid columns properly

**Comment by Shubham-1105** [comment]
@salil-naik corrected the structure but getting issue in removing padding
Screenshot (462)
Screenshot (463)

**Review by salil-naik — APPROVED** [review]

**Pull request merged**